apps/studio/src/server/modules/user/user.router.ts

```diff
@@ -56,6 +56,19 @@ export const userRouter = router({
         action: "manage",
       })
 
+      const possibleActor = await db
+        .selectFrom("User")
+        .where("id", "=", ctx.user.id)
+        .selectAll()
+        .executeTakeFirstOrThrow(
+          () =>
+            new TRPCError({
+              code: "NOT_FOUND",
+              message: "User not found",
+            }),
+        )
+      const actorName = possibleActor.name || possibleActor.email
```

Comment:
suggestion + nit: might wanna extract this logic out since it's duplicated

Reply:
Not doing now, cos it adds an additional layer of abstraction which would be harder to read through the code at quick glance. If it's duplicated multiple times then yeah we can explore the abstraction.